apps/src/gamelab/GameLab.js

```diff
@@ -1075,30 +1062,13 @@ GameLab.prototype.onP5Draw = function () {
  * for long enough and we have not done so already.
  */
 GameLab.prototype.captureInitialImage = function () {
-  if (!project.isOwner() || this.initialCaptureComplete || this.tickCount < CAPTURE_TICK_COUNT) {
+  if (this.initialCaptureComplete || this.tickCount < CAPTURE_TICK_COUNT) {
     return;
   }
   this.initialCaptureComplete = true;
-  captureImage();
+  captureThumbnailFromCanvas(document.getElementById('defaultCanvas0'));
```

Comment:
That's not quite right -- captureThumbnailFromElement takes `minCaptureIntervalMs`, which is the min time between captures. `captureThumbnailFromElement` and `captureThumbnailFromCanvas` both let the caller decide how many ticks to wait before attempting to capture the thumbnail.

That said, a major difference is that `captureThumbnailFromCanvas` is that we are not enforcing a min capture interval before calling it. We probably should have a minimum, since even though the capture itself is cheap, the network traffic to send it to the server is just as expensive. I've added the same min capture interval for artist and gamelab, allowing me to simplify the code related to this a bunch.